atomix/cluster/src/main/java/io/atomix/raft/zeebe/ZeebeLogAppender.java

```diff
@@ -16,16 +16,15 @@
 package io.atomix.raft.zeebe;
 
 import io.atomix.raft.storage.log.IndexedRaftLogEntry;
+import io.camunda.zeebe.util.buffer.BufferWriter;
 import java.nio.ByteBuffer;
 
 /**
  * A log appender provides a central entry point to append to the local Raft log such that it is
  * automatically replicated and eventually committed, and the ability for callers to be notified of
  * various events, e.g. {@link AppendListener#onCommit(IndexedRaftLogEntry)}.
  */
-@FunctionalInterface
```

Comment:
:cry: 

:wrench: We could still have a default implementation that gets overridden, no?